templates/ethstore.html

```diff
@@ -0,0 +1,524 @@
+		if (el && el.nodeName == "H1") {
+			var next = el.nextElementSibling
+			while (next && next.nodeName != "H1") {
+				if (next.nodeName == "BUTTON") {
+					click(next)
+				}
+				next = next.nextElementSibling
+			}
+
+		} else if (el && el.nodeName == "BUTTON") {
+			click(el)
+		}
+	}
+	Highcharts.stockChart('income-chart', {
+		rangeSelector: {
+			enabled: false
+		},
+		scrollbar: {
+			enabled: false
+		},
+		navigator: {
+			enabled: true
+		},
+		chart: {
+			type: 'column',
+			backgroundColor: "rgba(0,0,0,0.0)",
+		},
+		legend: {
+			enabled: true,
+			legend: {
+        enabled: true,
+        itemHoverStyle: {
+          color: 'var(--body-color)'
+        },
+        itemHiddenStyle: {
+          color: 'var(--shadow-light)'
+        }
+		  },
+		},
+		xAxis: {
+			range: 32 * 24 * 60 * 60 * 1000,
+			type: 'datetime',
+			crosshair: true
+		},
+		tooltip: {
+			style: {
+				color: 'var(--body-color)',
+				fontSize: '120%',
+				lineHeight: '22px',
+				fontFamily: "'Inter', sans-serif",
+			},
+			backgroundColor: 'var(--bg-color-nav-pool)',
+			borderColor: 'var(--shadow-light)',
+			shared: true,
+			useHTML: true,
+
+				split: false,
+			formatter: function () {
+				var add = ``
+
+				for (var i = 0; i < this.points.length; i++) {
+					if (this.points[i].series.name == "Total Staking Rewards") {
+						add += `<span style="color:${this.points[i].series.color};display: inline-block; width: 200px;">${this.points[i].series.name}: </span><b>${Highcharts.numberFormat(this.points[i].y, 6)} ETH</b><br/>`
+					} else if (this.points[i].series.name == "ETH.STORE") {
+						add += `<span style="color: ${this.points[i].series.color}; display: inline-block; width: 200px;">${this.points[i].series.name}: </span><b>${this.points[i].y.toFixed(3)} % p.a.</b><br/>`
+					} else {
+						add += `<span style="color: ${this.points[i].series.color}; display: inline-block; width: 200px;">${this.points[i].series.name}: </span><b>${Highcharts.numberFormat(this.points[i].y, 0)} ETH</b><br/>`
+					}
+				}
+				return `${new Intl.DateTimeFormat(navigator.language || 'en-US', { dateStyle: 'long' }).format(new Date(this.x))}<br/>` + add
+			}
+		},
+		yAxis: [{
+			title: {
+				text: 'Total Staking Rewards (ETH)'
+			},
+			opposite: false,
+			labels: {
+				formatter: function () {
+					return this.value.toFixed(0)
+				},
+			},
+		},
+		{
+			title: {
+				text: 'ETH.STORE (% p.a.)'
+			},
+			opposite: false,
+			labels: {
+				formatter: function () {
+					return (this.value).toFixed(1) + '%'
+				},
+			},
+			opposite: true,
+		},
+		{
+			title: {
+				enabled: false,
+				text: 'Total Effective Balance'
+			},
+
+			opposite: false,
+			labels: {
+				enabled: false,
+				formatter: function () {
+					return (this.value / 1000000).toFixed(0) + "M Ether"
+				},
+			},
+			opposite: true,
+		},
+		],
+		navigator: {
+			series: {
+				type: 'line'
+			}
+		},
+		series: [
+			{
+				name: 'Total Staking Rewards',
```

Comment:
https://staking.ethermine.org/statistics labels this "Total Staking Rewards (ETH)" i.e. adds "(ETH)" at the end. Should the same be done here?

Reply:
In my opinion it is unnecessary to specify the unit when every datapoint has "X ETH" as its tooltip, @Buttaa thoughts?

Reply:
FYI, something similar also applies to the chart's tooltip: The explorer now shows something like "Total Staking Reward: 212 ETH" while https://staking.ethermine.org/statistics shows something like "Total Staking Reward (ETH): 212 ETH" (the latter is worse in my humble opinion).

Reply:
Yes, this is exactly what I meant. Every datapoint on the chart already specifies the unit, which is why I deem the "(ETH)" unnecessary. In the same vein, the total effective balance is also denominated in ETH but doesn't specify its unit.

Reply:
I totally understand your reasoning and I am not saying that anything is wrong. It is just different to the EMS.
Maybe @Buttaa can let us know how it should be.

Reply:
After talking to him privately this change has been approved